tabbycat/breakqual/models.py

```diff
@@ -91,17 +93,19 @@ class BreakingTeam(models.Model):
     REMARK_DISQUALIFIED = 'd'
     REMARK_LOST_COIN_TOSS = 't'
     REMARK_WITHDRAWN = 'w'
+    REMARK_RESERVE = 'R'
     REMARK_CHOICES = (
         (REMARK_CAPPED, _("Capped")),
         (REMARK_INELIGIBLE, _("Ineligible")),
         (REMARK_DIFFERENT_BREAK, _("Different break")),
         (REMARK_DISQUALIFIED, _("Disqualified")),
         (REMARK_LOST_COIN_TOSS, _("Lost coin toss")),
         (REMARK_WITHDRAWN, _("Withdrawn")),
+        (REMARK_RESERVE, _("Reserve")),
```

Comment:
This is a related ask, but could we convert these into a `models.TextChoice` enum? I've been doing that when adding new items to an old-style choices list.

Could also be good to add an "Absent" remark.